src/views/portal/components/VerticleTimeline.tsx

```diff
@@ -0,0 +1,64 @@
+import ListGroup from "react-bootstrap/ListGroup";
+import styles from "views/portal/Portal.module.scss";
+
+import { Form } from "react-bootstrap";
+
+interface TimelineProps {
+	status: string | null;
+	application_submitted_date: string | null;
+	verdict_submitted_date?: string | null;
+	forms_submitted_date?: string | null;
+}
+
+function VerticleTimeline(props: TimelineProps) {
+	const first_item = (
+		<ListGroup.Item className={styles.list_item}>
+			<Form.Check disabled={true} checked={true} className={styles.check} />
+			Application submitted: {props.application_submitted_date}
+		</ListGroup.Item>
+	);
+
+	const second_item =
+		props.status === "submitted" ? (
```

Comment:
I'm not sure nested ternary operations are good practice, though looking online, the issue seems split. @taesungh How do you feel about this?